### PR TITLE
Fix getOwnerBalance order params order to ensure consistency

### PR DESCRIPTION
**Pull request by felix2feng**

### CODE REVIEW

File: contracts/core/Vault.sol [hunk 1]
```diff
@@ -172,12 +172,13 @@ contract Vault is
...
-     * @param  _owner           The address of the token owner
      * @param  _token           The address of the ERC20 token
+     * @param  _owner           The address of the token owner
      */
     function getOwnerBalance(
-        address _owner,
-        address _token
+        address _token,
+        address _owner
+
```
Comment (justinkchen) [inline contracts/core/Vault.sol]: remove extra line

**Comment by justinkchen** [comment]
wondering if we should also just change the interfaces for `incrementTokenOwner` and `decrementTokenOwner` too maybe? so its consistent within file?

**Comment by justinkchen** [comment]
also don't forget to we'll need to modify the contract generated wrapper in setprotocol.js as well when we integrate with this change. (and we'll need to `yarn deploy:development`)

**Comment by felix2feng** [comment]
Yeah - I think changing the interfaces  for `incrementTokenOwner` and `decreaseTokenOwner` makes sense.

**Pull request merged**